samples/amber.cc

```diff
@@ -480,6 +540,29 @@ int main(int argc, const char** argv) {
       // give clues as to the failure.
     }
 
+    // Dump the shader assembly
+    if (!options.shader_filename.empty()) {
+#if AMBER_ENABLE_SPIRV_TOOLS
```

Comment:
Makes me wonder whether you should just write the binary by default and have a second argument to control disassembly.

Reply:
Writing binary is tricker as I have to write to N files. Currently I write all shaders to a single file ....

Reply:
Right, the only other suggestion I have would be that -w specified a base file name and you output: <base>-<shader name>.[spv/svpasm]. At the same time this isn't a huge deal so feel free not to do it.

Reply:
Right, but in the .spv case, you'd have to write out N files because you can have multiple shader,pipeline pairs. Right now, it just writes them all into a single file.

It's doable, I'm just not sure if it's worth it. Is the non-asm version of this _useful_?

Reply:
Probably not enough to warrant the headaches.